options/locale/locale_en-US.ini

```diff
@@ -61,103 +61,103 @@ cancel = Cancel
 
 [install]
 install = Installation
-title = Initial configuration
-docker_helper = If you are running Gitea inside Docker, please read the <a target="_blank" rel="noopener" href="%s">guidelines</a> carefully before changing anything on this page.
-requite_db_desc = Gitea requires MySQL, MSSQL, PostgreSQL, SQLite3, or TiDB.
+title = Initial Configuration
+docker_helper = If you run Gitea inside Docker, please read the <a target="_blank" rel="noopener" href="%s">documentation</a> before changing any settings.
+requite_db_desc = Gitea requires MySQL, PostgreSQL, MSSQL, SQLite3 or TiDB.
 db_title = Database Settings
 db_type = Database Type
 host = Host
-user = User
+user = Username
 password = Password
 db_name = Database Name
-db_helper = Please use the INNODB engine with utf8_general_ci charset for MySQL.
-ssl_mode = SSL Mode
+db_helper = Note to MySQL users: please use the InnoDB storage engine and the 'utf8_general_ci' character set.
+ssl_mode = SSL
 path = Path
-sqlite_helper = The file path to the SQLite3 or TiDB database. <br>Please use the absolute path when you start as service.
-err_empty_db_path = SQLite3 or TiDB database path cannot be empty.
-err_invalid_tidb_name = TiDB database name does not allow characters "." and "-".
-no_admin_and_disable_registration = You cannot disable registration without creating an admin account.
-err_empty_admin_password = Admin password cannot be empty.
-
-general_title = General Application Settings
-app_name = Application Name
-app_name_helper = You can put your organization name here.
+sqlite_helper = File path for the SQLite3 or TiDB database.<br>Enter an absolute path if you run Gitea as a service.
+err_empty_db_path = The SQLite3 or TiDB database path cannot be empty.
+err_invalid_tidb_name = The TiDB database name cannot contain the '.' and '-' characters.
+no_admin_and_disable_registration = You cannot disable user self-registration without creating an administrator account.
+err_empty_admin_password = The administrator password cannot be empty.
+
+general_title = General Settings
+app_name = Site Title
+app_name_helper = Enter your group name here.
```

Comment:
... uhmmm, no, I'd rather actually stick to organization here. Here it actually means the company that is hosting the gitea instance, so it makes sense to use organization.

Reply:
Oops… that's an overzealous search-and-replace. Agree this should be 'organization' or 'company' or something. Maybe not 'organization' to prevent cognitive overlap with what we now call a 'group'.